docs/source/developer/besu-ansible.md

```diff
@@ -0,0 +1,121 @@
+# hyperledger-besu Configurations
+
+In the Blockchain Automation Framework project, ansible is used to automate the certificate generation, put them in the vault and generate value files, which are then pushed to the repository for deployment, using GitOps. This is achieved using Ansible playbooks.
+Ansible playbooks contains a series of roles and tasks which run in sequential order to achieve the automation.
+
+```
+/hyperledger-besu
+|-- charts
+|   |-- node_orion
+|-- images
+|-- configurations
+|   |-- roles/
+|   |-- samples/
+|   |-- deploy-network.yaml
+|-- releases
+|   |-- dev/
+|-- scripts
+```
+
+For Hyperledger Besu, the ansible roles nad playbooks are located at `/platforms/hyperledger-besu/configuration/`. Some of the common roles and playbooks between hyperledger-besu, Hyperledger-Indy, Hyperledger-Fabirc, Hyperledger-Besu and R3-Corda are located at `/platforms/shared/configurations/`
```

Comment:
"roles and .." Check the complete sentence for typos

Reply:
"hyperledger-besu, Hyperledger-Indy, Hyperledger-Fabirc, Hyperledger-Besu and R3-Corda" duplicate Besu, Fabric spelling wrong.